### ISSUE TITLE
Expose neo4j-contrib/neo4j-helm as helm repository

### ISSUE THREAD

**Issue by aitorhh** [neo4j-contrib/neo4j-helm#16]
I believe it would be good to add the deployment of the Charts to github pages or release page. Several options could be used:

- Automatically: https://github.com/helm/chart-releaser/ 
- Manually: "helm package . -d .deploy/; helm repo index <gh-pages-link> .deploy/"

Both, can be use in a CI/CD pipeline.

**Comment by moxious** [comment]
Thanks for this suggestion.   I think it's a good one and I will take it on.  Sorry for the tardy response - I'll get to this when I can.

**Comment by moxious** [comment]
Some updates on this; the tooling isn't integrated yet, but the repository index.yaml file was moved to the master branch, and [Helm Hub](https://hub.helm.sh/charts/neo4j/neo4j) is showing all recent releases ATM

**Comment by moxious** [comment]
Still needing to follow up on this.  Additional report:

```
[user@localhost ~]$ helm repo add stable https://github.com/neo4j-contrib/neo4j-helm
Error: looks like "https://github.com/neo4j-contrib/neo4j-helm" is not a valid chart repository or cannot be reached: failed to fetch https://github.com/neo4j-contrib/neo4j-helm/index.yaml : 404 Not Found 
[user@localhost ~]$ curl -O https://github.com/neo4j-contrib/neo4j-helm/blob/master/index.yaml
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current 
                                 Dload  Upload   Total   Spent    Left  Speed 
100  107k    0  107k    0     0   206k      0 --:--:-- --:--:-- --:--:--  206k 
[user@localhost ~]$ helm repo add stable https://github.com/neo4j-contrib/blob/master/neo4j-helm
Error: looks like "https://github.com/neo4j-contrib/blob/master/neo4j-helm" is not a valid chart repository or cannot be reached: failed to fetch https://github.com/neo4j-contrib/blob/master/neo4j-helm/index.yaml: 404 Not Found 
```

**Comment by moxious** [comment]
Current status here is I have an index.yaml file to act as a repo location, but chart-releaser is really not cooperating and I think suffering from some docs unclarities that make it hard to follow.  Namely it seems to be expecting that my tag structure and the name of the releases always match, and they don't - and also that the tgz files are hosted in the repo themselves.  Additionally, I'd like this repo to be self-hosted, and not have the "split" that chart releaser is assuming, where there's a "repository github repo" and a "chart repo".

So I'm looking into this to try to resolve it, but really struggling with silly bits of helm & cr to make this work.   Help solicited.

**Comment by jennyowen** [comment]
@moxious It's interesting to see how aws does this. For example, https://github.com/aws/eks-charts
releases everything in a big flat repository on the `gh-pages` branch, so that the github.io page serves it in a way that helm repositories expect :  https://github.com/aws/eks-charts/tree/gh-pages
It's quite a neat solution!

I'm not familiar with the `cr` tool, but would releasing like that mean we don't need it?
I'll try and prototype something.

**Comment by moxious** [comment]
@jennyowen I think in the end they're equivalent solutions really.   The cr tool in the end is just trying to help automate what is the same thing:  you place certain yaml files in certain places, make them web accessible, and then poof, you're a helm repo.  That's basically it.   I just burned a fair bit of time failing to figure out the cr tool.  *Any* reasonably automatable solution which makes helm happy, and which can be maintained over time is ultimately fine.

**Comment by lukaszo** [comment]
@moxious this command worked just fine: 
```bash 
helm repo add neo4j-helm https://neo4j-contrib.github.io/neo4j-helm/
```

I've found the correct command here: https://artifacthub.io/packages/helm/neo4j-helm/neo4j?modal=install
